interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -41,9 +58,55 @@ components:
           type: string
           example: v0.0.1-445ad3
 
+    InjectGeozonesRequest:
+      type: object
+      properties:
+        source:
+          anyOf:
+            - $ref: '#/components/schemas/GeozonesHttpsSource'
+
+    GeozonesHttpsSource:
+      type: object
+      required:
+        - url
+      properties:
+        url:
+          type: string
+          format: url
+          description: The URL at which the geo-awareness information shall be downloaded from.
+          example: https://caa.example.com/geozones.json
+
+    InjectGeozonesResponse:
+      type: object
+      required:
+        - result
+      properties:
+        result:
+          description: >-
+            The status of the Geozones dataset import.
+
+            - `Activating`: the USS is processing the request and is currently activating the dataset.
+
+            - `Ready`: the Geozones dataset has been successfully imported and the USS can receive test requests.
+            
+            - `Deleting`: the Geozones dataset is being deactivated and deleted.
+            
+            - `Rejected`: the Geozones dataset was rejected because the dataset is invalid.
+            
+            - `Error`: the Geozones dataset activation or deactivation failed. The description field is required in this case.
+          type: string
+          enum: [Starting, Ready]
+          example: Ready
+        description:
```

Comment:
Personally, I would recommend `message`, but not important

Reply:
Sounds good.